app/components/Notifications/index.js

```diff
@@ -143,11 +151,15 @@ const Notifications = ({
         indexToStop,
       ]);
 
-      if (!_isEqual(union, readNotifications)) {
+      /*
+      * TODO: Fix bug with reading notifications, information in Notification center and Dropdown
+      * may vary if notifications are received on the notifications page
+      */
+      /*if (!_isEqual(union, readNotifications) && !document.hidden) {
```

Comment:
remove dead code

Reply:
I think, this commented code is necessary for understanding of problem